plugins/modules/zos_job_submit.py

```diff
@@ -230,12 +230,36 @@
              For JCL ERRORs, this will be None.
           type: int
           sample: 00
+        steps:
+          description:
+            Series of JCL steps that were executed and their return codes.
+          type: list
+          elements: dict
+          contains:
+            stepid:
+              description:
+                Name of the step shown as "was executed" in the DD section.
+              type: str
+              sample: "STEP0001"
+            stepcc:
+              description:
+                Actual CC returned for this step in the DD section.
```

Comment:
See minor comment about the use of "Actual" in other comments

Reply:
Done